gen/calc.py

```diff
@@ -1319,7 +1383,20 @@ def calculate_fault_domain_detect_contents(fault_domain_detect_filename):
         'enable_mesos_ipv6_discovery': 'false',
         'log_offers': 'true',
         'mesos_cni_root_dir_persist': 'false',
-        'enable_mesos_input_plugin': 'true'
+        'enable_mesos_input_plugin': 'true',
+        'calico_enabled': 'true',
+        'calico_network_cidr': '192.168.0.0/16',
```

Comment:
We may track this in another PR if you insist because this mandatory setting will fail new clusters created by like ccm, and dcos-e2e.

Reply:
Tracked here: https://jira.mesosphere.com/browse/DCOS-60734

Reply:
I'm afraid we may deprioritize this task even when it's mandatory and must be done in the same release. We had such thing few times previously, it's not fun at all and I don't see why we should split this task. We have plenty of time to resolve the cidr question in this PR. I won't block it, but would highly recommend not to postpone that.

Reply:
Understood. I wanna make it in anothe PR because we may need to have this parameter first before we can make it available in dcos-e2e, dcos-launch( and ccm based on it), terraform. 
Anyway, I will do research on the repo and project owner to have their ideas.